api/src/main/java/io/opentelemetry/OpenTelemetry.java

```diff
@@ -116,12 +126,39 @@ private static OpenTelemetry getInstance() {
   }
 
   private OpenTelemetry() {
-    TracerProvider tracerProvider = loadSpi(TracerProvider.class);
-    tracer = tracerProvider != null ? tracerProvider.create() : DefaultTracer.getInstance();
-    MeterProvider meterProvider = loadSpi(MeterProvider.class);
-    meter = meterProvider != null ? meterProvider.create() : DefaultMeter.getInstance();
-    TaggerProvider taggerProvider = loadSpi(TaggerProvider.class);
-    tagger = taggerProvider != null ? taggerProvider.create() : DefaultTagger.getInstance();
+    TracerProvider tracerProvider =
+        loadSpi(TracerProvider.class, Thread.currentThread().getContextClassLoader());
```

Comment:
`Thread.currentThread().getContextClassLoader()` is not surrounded in a `doPrivileged()` block.  This may cause Java 2 security exception.